### PR TITLE
Avoid GUID collisions

### PR DESCRIPTION
**Pull request by JoshLove-msft**

### CODE REVIEW

**Review by pakrym — APPROVED** [review]

File: eng/pipelines/templates/jobs/archetype-sdk-tests.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ jobs:
           packageType: sdk
           version: "$(DotNetCoreSDKVersion)"
 
-      - script: dotnet test eng/service.proj --framework $(TestTargetFramework) --logger "trx;LogFileName=$(TestTargetFramework).trx" --logger:"console;verbosity=normal" /p:ServiceDirectory=${{ parameters.ServiceDirectory }} /p:IncludeSrc=false /p:IncludeSamples=false
+      - script: dotnet test eng/service.proj --framework $(TestTargetFramework) --logger "trx;LogFileName=$(TestTargetFramework).trx" --logger:"console;verbosity=normal" /p:ServiceDirectory=${{ parameters.ServiceDirectory }} /p:IncludeSrc=false /p:IncludeSamples=false /p:BuildInParallel=false
```
Comment (pakrym) [inline eng/pipelines/templates/jobs/archetype-sdk-tests.yml]: Don't know about this but the random part is fine.

**Review by mikeharder — APPROVED** [review]

**Pull request merged**